src/meta-srv/src/failure_detector.rs

```diff
@@ -14,6 +14,8 @@
 
 use std::collections::VecDeque;
 
+use serde::{Deserialize, Serialize};
+
 /// This is our port of Akka's "[PhiAccrualFailureDetector](https://github.com/akka/akka/blob/main/akka-remote/src/main/scala/akka/remote/PhiAccrualFailureDetector.scala)"
```

Comment:
BTW, Akka has switched to BuSL 1.1 which is not an open-source license. And code translation is still under the rick of being a derivative work.

So either specify the tag of version still released under Apache License 2.0, or try to use a link to https://github.com/apache/incubator-pekko/blob/main/remote/src/main/scala/org/apache/pekko/remote/PhiAccrualFailureDetector.scala.

Reply:
That's great to know, thank you for pointing it out. Would you kind enough to change it in this pr? We can link to the code in akka version 2.6:
https://github.com/akka/akka/blob/v2.6.21/akka-remote/src/main/scala/akka/remote/PhiAccrualFailureDetector.scala

Reply:
OK. Let's do it in another PR. I can preapre one of course.

Reply:
Here it is - https://github.com/GreptimeTeam/greptimedb/pull/2714